src/main/java/com/alexbro/onlinebank/facade/validation/ValidationConstants.java

```diff
@@ -0,0 +1,13 @@
+package com.alexbro.onlinebank.facade.validation;
+
+public interface ValidationConstants {
+
+    String NAME_REGEXP = "^((?![0-9]).)*$";
```

Comment:
Separate regexps and messages. Add 2 inner classes for them same as in `WebConstants`